ruby/files_and_serialization/files_and_serialization.md

```diff
@@ -27,7 +27,7 @@ Look through these now and then use them to test yourself after doing the assign
   2. Watch [icc0612's introduction to serialization](https://www.youtube.com/watch?v=uS37TujnLRw). It will explain the concept of serialization before you implement it in Ruby.
   3. Read [Choosing the Right Serialization Format](https://www.sitepoint.com/choosing-right-serialization-format/) for more information about the various serialization options you can choose from.
   4. Skim [Alan Skorkin's](http://www.skorks.com/2010/04/serializing-and-deserializing-objects-with-ruby/) post about serialization. Some of the code examples are outdated, but this post still contains relevant information. If you want to follow along with the code examples, copy and paste them into your code editor and use word wrap.
-  5. Read the [Chapter on File I/O](http://ruby.bastardsbook.com/chapters/io/) of the Bastard's Book of Ruby.
+  5. Read the [Chapter on File I/O](http://ruby.bastardsbook.com/chapters/io/) of the Bastard's Book of Ruby. (Note: the ruby gem "open-URI" referenced in this assignment has been updated since publishing this material.  calling open(filename) is deprecated as of Ruby 3.0+. The updated usage is now to call URI.open(filename) directly)
```

Comment:
```suggestion
  5. Read the [Chapter on File I/O](http://ruby.bastardsbook.com/chapters/io/) of the Bastard's Book of Ruby. Note: the ruby gem "open-URI" referenced in this assignment has been updated since publishing this material.  Calling `open(filename)` is deprecated as of Ruby 3.0+. The updated usage is now to call `URI.open(filename)` directly.
```

@DNeel10 This is pretty much perfect, I just have a few suggestions for minor styling and formatting.

Reply:
Thanks, @ChargrilledChook .  At this point in the process, is there a way to accept the suggested changes within github, or is it still more appropriate for me to make the edits myself (same as before) and then push them again via git push in the command line?

Reply:
No worries @DNeel10.

> . At this point in the process, is there a way to accept the suggested changes within github, or is it still more appropriate for me to make the edits myself (same as before) and then push them again via git push in the command line?

There's a few options - if you're happy with my suggestion, I think you should be able to click a button to accept it (although I'm unsure how it looks on your end). On my end I can see 'commit suggestion'. If you can't see that, I'm happy to do it for you.

![image](https://user-images.githubusercontent.com/58506115/200113153-bdbf23c2-b4b6-4ea8-b866-9d76dec9adcd.png)

Redoing it locally and re-pushing is also a totally fine option - the PR will automatically update with any changes you make.

Finally if it's easier, it is possible to edit files directly from GitHub. On the 'files changed' tab, it's possile to edit the file directly.

![image](https://user-images.githubusercontent.com/58506115/200113337-6cc2ee05-0837-4e60-b65c-5749f1d0bdd1.png)

Pushing from your local is probably the closest thing to a 'real' workflow and could be good practice, but for small change like this any of these options are completely fine.